testing/test_rx_tx/test_rx_tx.py

```diff
@@ -0,0 +1,388 @@
+import numpy as np
+import uhd.usrp as usrp
+import uhd
+import math
+import cmath
+import sys
+import time
+import threading
+
+# Define constants
+ADDR = "num_recv_frames=512,num_send_frames=256,send_buff_size=2304000"
```

Comment:
These are not actually addresses, but actually extra device args.

Reply:
Could use a better name

Reply:
What about DEV_ARGS?